src/main/java/duke/DukeException.java

```diff
@@ -0,0 +1,11 @@
+package duke;
+
+/** Represents the errors that can occur
+ * when the Duke chatbot is running.
+ */
+public class DukeException extends Exception {
+    public DukeException(String s) {
+        super(s);
```

Comment:
it would be nice if you can actually have more specific exceptions that Duke might give